context/app/static/js/components/home/ExploreTools/ExploreTools.tsx

```diff
@@ -23,7 +23,7 @@ const workspaceCTAGuest = {
 const cards = [
   {
     title: 'Analyze data in Workspaces',
-    icon: <entityIconMap.Workspace color={theme.palette.info.main} fontSize="1.5rem" />,
```

Comment:
The icon changes allowed for the earlier fix to be reverted?

Reply:
Yes - with the `createSvgIcon` util, theme values like `info` are properly recognized, so the actual value doesn't need to be passed in anymore. This should help avoid annoying related regressions in the future.